source/fundamentals/data-formats/serialization.txt

```diff
@@ -0,0 +1,190 @@
+.. _fundamentals-kotlin-serialization:
+
+====================
+Kotlin Serialization
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+The Kotlin driver supports the ``kotlinx.serialization`` library for serializing 
+and deserializing Kotlin objects. 
+
+The driver provides an efficient ``Bson`` serializer that can can be used with 
+``@Serializable`` classes to handle the serialization of Kotlin objects to BSON data. 
+The ``bson-kotlinx`` library also supports :ref:`custom codecs <kotlin-custom-codec>`
+with configurations such as encoding defaults, encoding nulls, and defining class 
+discriminators.
+
+.. note::
+
+    To use the ``Codec`` interface to specify custom encoding and decoding of 
+    Kotlin objects to BSON data, see the :ref:`Codecs <fundamentals-codecs>` 
+    documentation.
+
+Although you can use the Kotlin driver with the Kotlin Serialization's ``Json`` 
+library, the ``Json`` serializer does *not* directly support BSON value types such
```

Comment:
Question: what is "Kotlin Serialization"? Why is Serialization capitalized?

Reply:
it's referring specifically to the official framework. I've updated for clarity